src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/FilingsService.java

```diff
@@ -249,6 +249,7 @@ private void setUpdateSubmissionData(
       data.put(CHANGES_FIELD, updateSubmission.getChanges());
       data.put(ADDITIONS_FIELD, updateSubmission.getAdditions());
       data.put(CESSATIONS_FIELD, updateSubmission.getCessations());
```

Comment:
if (updateSubmission.getTrustAdditions() != null && ! updateSubmission.getTrustAdditions().isEmpty) {
     data.put(TRUST_DATA, updateSubmission.getTrustAdditions());
}

Reply:
Still think we should add only if not null.

Reply:
pushed a change